tasks/cleanup.yml

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: MIT
+- name: Remove snapshots from previous runs
```

```suggestion
---
- name: Remove snapshots from previous runs
```
ansible-lint requires the `---` YAML document start marker - but *not* the document end marker . . .